app/helpers/housing_costs_helper.rb

```diff
@@ -0,0 +1,8 @@
+module HousingCostsHelper
+  def outgoings_links(level_of_help)
```

Comment:
As mentioned in Slack, this name clashes with the method defined in OutgoingsHelper. Each method name in `helpers` needs to be unique.